enable_location.sh

```diff
@@ -25,8 +25,13 @@ ERROR_REDIRECT_CODES=$(get_id_var ${LOCATION_ID} ERROR_REDIRECT_CODES)
 ENABLE_WEB_SOCKETS=$(get_id_var ${LOCATION_ID} ENABLE_WEB_SOCKETS)
 ADD_NGINX_LOCATION_CFG=$(get_id_var ${LOCATION_ID} ADD_NGINX_LOCATION_CFG)
 
+# Backwards compatability
+if [ "`echo ${PROXY_SERVICE_HOST} | grep -i 'https\?://'`" = "" ]; then
```

Comment:
Im not sure I like it all as it assumes http or https what if you proxied gopher or ftp ... 
i think we can just detect :// being missing although maybe this should just be another flag  ?

PROXY_HOST_INCLUDES_PROTOCOL=true ?

Thousght ?